skeleton/language/en/common.php.twig

```diff
@@ -23,11 +23,11 @@ $lang = array_merge($lang, array(
 	'DEMO_HELLO'		=> 'Hello %s!',
 	'DEMO_GOODBYE'		=> 'Goodbye %s!',
 
-	'ACP_DEMO'					=> 'Settings',
-	'ACP_DEMO_GOODBYE'			=> 'Should say goodbye?',
-	'ACP_DEMO_SETTING_SAVED'	=> 'Settings have been saved successfully!',
+	'ACP_{{ EXTENSION.extension_name|upper }}'					=> 'Settings',
+	'ACP_{{ EXTENSION.extension_name|upper }}_GOODBYE'			=> 'Should say goodbye?',
+	'ACP_{{ EXTENSION.extension_name|upper }}_SETTING_SAVED'	=> 'Settings have been saved successfully!',
 
-	'ACME_DEMO_NOTIFICATION'	=> 'Acme demo notification',
+	'{{ EXTENSION.vendor_name|upper }}_{{ EXTENSION.extension_name|upper }}_NOTIFICATION'	=> 'Acme demo notification',
 
-	'VIEWING_ACME_DEMO'			=> 'Viewing Acme Demo',
+	'VIEWING_ACME_DEMO'			=> 'Viewing {{ EXTENSION.extension_display_name|capitalize }} page',
```

Comment:
Capitalize not needed here, since the display name should in theory be the written the way the author wants the it to look. We may, however, want to use the twig escape filter here instead.

Reply:
Correct on the Capitalize.
Is the `|e` filter sufficient?